src/main/java/net/consensys/orion/cmd/Orion.java

```diff
@@ -543,6 +532,28 @@ public void run(final Config config, final boolean clearKnownNodes) {
     isRunning.set(true);
   }
 
+  private static void createAndVerifyTLSCert(
+      final Path tlsKey,
+      final Path tlsCert,
+      final Optional<URL> nodeUrl,
+      final String label) {
+    try {
+      TLS.createSelfSignedCertificateIfMissing(tlsKey, tlsCert, nodeUrl);
```

Comment:
The return value indicates if the cert was created or not, thinking it makes to check this as well? Or can we just rely on the exception?

Reply:
I've updated the return value of `TLS.createSelfSignedCertificateIfMissing`, it was misleading.